core-bundle/contao/classes/DataContainer.php

```diff
@@ -161,6 +161,21 @@ abstract class DataContainer extends Backend
 	 */
 	public const SORT_BOTH = 18;
 
+	/**
+	 * Primary database index
+	 */
+	public const INDEX_PRIMARY = 'primary';
+
+	/**
+	 * Regular database index
+	 */
+	public const INDEX_REGULAR = 'index';
```

Comment:
```suggestion
	public const INDEX_SECONDARY = 'index';
```
AFAIK “secondary index” is a widely used term for describing a database index that is not the primary key.